android/project.json

```diff
@@ -16,7 +16,9 @@
     ],
     "excludes": [
         ".*",
-        "xliffs"
+        "xliffs",
+    	"res",
+    	"node_modules"
```

Comment:
node_modules is already added in exclude list from loctool. no need to write additionally.

Reply:
fixed